### ISSUE TITLE
Tracker for debug reporting information to work correctly across the board.

### ISSUE THREAD

**Issue by TimJSwan89** [runtimeverification/rv-match_testing#9]
Currently, the project successfully parses some `k-bin-to-text` files automatically as well as gets the tail of make build output and puts those into the test results when a build fails. 

However, this isn't happening very consistently. Some projects have useful information that isn't getting reported. Issue #7 is a predecessor to this one. Even when that issue is closed, there may be more work on ensuring that information is getting reported properly.

The following criterion should be fulfilled before this issue is closed:
For both `status.sh` and `run_set.sh`:
1. They should report some useful debug information for all fails for projects in the test report.
2. Issue #7 should be closed.
3. All `k-bin-to-text` files should be reported as non-parsable, parsed and displayed, or reported as missing the expected tags like `<k>` when parsed.
4. All failed configure or make scripts should include that information in the test results including useful debug information like tails.

**Comment by TimJSwan89** [comment]
`status.sh` and `run_set.sh` merging is past the point of no return.

1. appears to be true, however, currently `kcc_config` results are posted under `no_kcc_config_generated` instead of `make` which is a problem because neither `regression` or `acceptance` publish that test. **This needs to be fixed.**

2. is complete.

3. has a solution documented in #7 essentially where one of the dropped `kcc_config`'s are inspected as described here when the default extract function is used and the one specified if the extract function is defined. I suppose the best option could be to simply just attempt to parse all of the dropped `kcc_config` files. The best functionality would be to state how many `kcc_config` files are in the project, where they each dropped, and each of their parse results. This will take time, but it is worth it for making the test suite useful and scalable. It should reduce frustrating headache work in the future with trying to track down which `kcc_config` file went where. That sounds like a nightmare. **This needs to be fixed.**

4. is considered fixed in practice, given that points 1 and 3 are met, since it was tested on `crashless.ini` which is nearly the entire set of projects.

In summary, **points 1 and 3 still have work to be done**.

**Comment by TimJSwan89** [comment]
To consider point 1 completed properly, string escaping regarding `<![CDATA[]]>` terms in the XML should be functioning properly. **This needs to be fixed.**

**Comment by TimJSwan89** [comment]
Update:
So the remaining **3 subpoints** to ensure are completed are:
1a. `k-bin-to-text` results included in the xml element with make fail info.
1b. `<![CDATA[]]>` string formatting is properly handled and escaped.
3. `kcc_config`s dropped in different locations auto-handled for each project. 

More detail on 3: The `_extract` function in `test.sh` scripts are deprecated since all interesting files can simply be prepended with `kcc_` and their locations found by the master scripts. Usually the simpler these scripts are, the better.

**Comment by TimJSwan89** [comment]
`rv-match_project_acceptance` build 5 and an local test with a bunch of `kcc_config`s confirms this is complete. Closing issue.
